ansible/roles/test/tasks/qos_config.yml

```diff
@@ -0,0 +1,46 @@
+### This test implements several test cases from the test plan:
+### https://github.com/Azure/SONiC/wiki/QoS-configuration-in-Config-DB.-ECN-WRED-configuration-utility-test-plan
+### It requires no external connections
+###
+### Usage example:
+###    ansible-playbook test_sonic.yml -i inventory --limit arc-switch1028 --tags qosconfig -e skip_interface_check=true
+
+- set_fact:
+    test_files_dir="roles/test/tasks/qos"
+    tmp_dir="/tmp"
+    test_wred_values="[ '491520', '516096', '430080' ]"
+
+  # Read and store original rmin value for AZURE_LOSSY
+- name: Get red_min_threshold
+  shell: ecnconfig -l | grep -A20 AZURE_LOSSY | grep red_min_threshold | awk '{print $2}'
+  register: wred_value
+  failed_when: wred_value.rc != 0
+
+- set_fact:
+    red_min_threshold={{ wred_value.stdout }}
+
+- block:
+    - name: Copy test files to DUT
+      copy: src={{ item }} dest={{ tmp_dir }}
+      with_fileglob:
+         - "{{ test_files_dir }}/*"
+  vars:
+    ansible_shell_type: docker
+    ansible_python_interpreter: docker exec -i database python
+
+- block:
+    # Test case #3(MA): Check configuration applied
+    - name: Check if ASIC DB configured qos parameters
+      shell: docker exec database redis-cli -n 1 KEYS ASIC_STATE:SAI_OBJECT_TYPE_QUEUE* | grep QUEUE
+      register: grep_queue
+      failed_when: grep_queue.rc != 0
```

Comment:
I am not sure if this is a big deal or not. But we do have some devices doesn't yet have qos config.

Failing test on a dut which doesn't have qos config seems to be too harsh?

Is there a way to early exit instead of failing the test?

Please don't spend too much time on this question. It is a very minor issue. I am ok with it not fixed.